src/test/java/com/wikia/webdriver/testcases/adstests/TestUrlBuilder.java

```diff
@@ -4,277 +4,383 @@
+                    (String) data[5]);
+            Assertion.assertEquals(new UrlBuilder("sandbox", (Boolean) data[2], (Boolean) data[3])
+                            .getUrlForPath((String) data[0], (String) data[1]),
+                    (String) data[6]);
+            Assertion.assertEquals(new UrlBuilder("sandbox-mercurydev", (Boolean) data[2], (Boolean) data[3])
+                            .getUrlForPath((String) data[0], (String) data[1]),
+                    (String) data[7]);
+            Assertion.assertEquals(new UrlBuilder("dev-dmytror", (Boolean) data[2], (Boolean) data[3])
+                            .getUrlForPath((String) data[0], (String) data[1]),
+                    (String) data[8]);
+            Assertion.assertEquals(new UrlBuilder("staging", (Boolean) data[2], (Boolean) data[3])
+                            .getUrlForPath((String) data[0], (String) data[1]),
+                    (String) data[9]);
+        }
+    }
+
+    @Test(groups = {"TestUrlBuilder", "unitTests"})
+    public void urlWithLanguageBuilderTests() {
+        Configuration.setTestValue("qs", "");
+
+        for (Object[] data : TEST_DATA_LANGUAGE) {
+            Assertion.assertEquals(new UrlBuilder("prod", (Boolean) data[2], (Boolean) data[3])
+                            .getUrlForPath((String) data[0], (String) data[10], (String) data[1]),
+                    (String) data[4]);
+            Assertion.assertEquals(new UrlBuilder("preview", (Boolean) data[2], (Boolean) data[3])
+                            .getUrlForPath((String) data[0], (String) data[10], (String) data[1]),
+                    (String) data[5]);
+            Assertion.assertEquals(new UrlBuilder("sandbox", (Boolean) data[2], (Boolean) data[3])
+                            .getUrlForPath((String) data[0], (String) data[10], (String) data[1]),
+                    (String) data[6]);
+            Assertion.assertEquals(new UrlBuilder("sandbox-mercurydev", (Boolean) data[2], (Boolean) data[3])
+                            .getUrlForPath((String) data[0], (String) data[10], (String) data[1]),
+                    (String) data[7]);
+            Assertion.assertEquals(new UrlBuilder("dev-dmytror", (Boolean) data[2], (Boolean) data[3])
+                            .getUrlForPath((String) data[0], (String) data[10], (String) data[1]),
+                    (String) data[8]);
+            Assertion.assertEquals(new UrlBuilder("staging", (Boolean) data[2], (Boolean) data[3])
+                            .getUrlForPath((String) data[0], (String) data[10], (String) data[1]),
+                    (String) data[9]);
+        }
+    }
+
+    @Test(groups = {"TestUrlBuilder", "unitTests"})
+    public void getWikiaUrlTests() {
+
+        Assertion.assertEquals(new UrlBuilder("preview", false).getWikiGlobalURL(),
+                "http://www.preview.wikia.com");
+        Assertion.assertEquals(new UrlBuilder("sandbox-s1", false).getWikiGlobalURL(),
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Define a constant instead of duplicating this literal "sandbox-s1" 4 times. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1192)